<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>JogWindow</class>
 <widget class="QWidget" name="JogWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="btnEMinus">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>50</y>
     <width>70</width>
     <height>70</height>
    </rect>
   </property>
   <property name="text">
    <string>E-</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnYPlus">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>50</y>
     <width>70</width>
     <height>70</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string>Y+</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnHomeAxis">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>400</y>
     <width>70</width>
     <height>70</height>
    </rect>
   </property>
   <property name="text">
    <string>Home Axis</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnXPlus">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>110</y>
     <width>70</width>
     <height>70</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>X+</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnEPlus">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>180</y>
     <width>70</width>
     <height>70</height>
    </rect>
   </property>
   <property name="text">
    <string>E+</string>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>410</y>
     <width>320</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="btnHalfStep">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>0.5</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnOneStep">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>1</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnTwoStep">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>2</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnFiveStep">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>5</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>390</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Steps per axis:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnYMinus">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>180</y>
     <width>70</width>
     <height>70</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string>Y-</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnZMinus">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>180</y>
     <width>70</width>
     <height>70</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string>Z-</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnXMinus">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>110</y>
     <width>70</width>
     <height>70</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string>X-</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnZPlus">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>70</width>
     <height>70</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string>Z+</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnClose">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>10</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
